lib/render/editor/editor.html

```diff
@@ -65,12 +65,50 @@
           }
         };
 
+        const setTrashPosition = () => {
+          var width = $('#blocklyDiv').width() - 68;
+          var height = $('#blocklyDiv').height() - 210;
+          $('.blocklyTrash').css("transform", `translate(${width}px, ${height}px)`);
+          $('.blocklyTrash').css("display", "unset");
+        };
+
+        var setTrashTimeout;
+
+        const localOnResize = () => {
```

Comment:
@flbulgarelli That's one of the nasty things i was talking about. That's a hack for ensuring that the function will run only when the event has finished and avoid running it several times when multiple resizes happen.

Reply:
`setTimeout` without a second parameter defaults to zero, which means that the function will be run as soon as it can. I think that the `clearTimeout` and the `setTrashTimeout` aren't needed.

Can we name this `asyncSetTrashPosition` or something like that?

Reply:
It's the only hack i found to ensure that the event will run only once the resize event has finished